### DISCUSSION TITLE
Filtered / blocked / muted keywords in results

### DISCUSSION THREAD

**Discussion by ghost** [searxng/searxng#2528] [Ideas]
Why isn't there an ability to remove results containing certain keywords, phrases, regex patterns etc. ?
Why is this not one of the core features?
I've seen some previous discussions on this but they didn't seem to go anywhere.
To me when I first learned about 'meta search engine' this was one of the very first exciting possibilities that popped into my mind about why I would want to use one, the view in my mind was that all these results from all these engines would be funnelled into a merged list of results and then sorted and filtered down according to my own rules to create the perfect tailored resultset but this final control seems to be completely lacking?

**Comment by unixfox** [comment]
Somewhat related issue: https://github.com/searxng/searxng/issues/1371
